bin/opt8n/src/opt8n.rs

```diff
@@ -183,6 +201,94 @@ impl Opt8n {
     }
 }
 
+// TODO: Consider adding `From` implementation for
```

Comment:
Agree - I will add issues in op-alloy for this.

Reply:
Sounds good, I can open a PR for these issues on `op-alloy` later today/tomorrow.